README.md

```diff
@@ -4,23 +4,24 @@
 
 Fully-featured GraphQL Server with focus on easy setup, performance & great developer experience
 
-[Docs](https://www.graphql-yoga.com/docs)
+[Docs](https://www.the-guild.dev/graphql/yoga-server/v3)
 
 ### Installation
 
 ```shell
-npm i @graphql-yoga/node graphql
+yarn add graphql-yoga graphql
 ```
 
 ### Quickstart
 
-You will need to provide schema to Yoga, either by an existing executable schema, or by providing your type definitions and resolver map:
+Make a schema, create Yoga and start a Node server with it:
 
 ```ts
-import { createServer } from '@graphql-yoga/node'
+import { createSchema, createYoga } from 'graphql-yoga'
+import { createServer } from 'http'
 
-const server = createServer({
-  schema: {
+const yoga = createYoga({
```

Comment:
This is tricky because v3 is the main branch right now and people will see this directly even if the latest version is v2.

Reply:
Hmm, but this belongs on v3. Maybe put back v2 as main until v3 is official?

Reply:
can we add `graphql-yoga@v3` on `npm` so on readme we can give code examples then link users to `v2`/`latest` docs on our site. We will make it clear that this readme is currently for alpha

Reply:
We need to get this released first then `v3` will be available.
Actually we can set `v2` back to main branch since I forced `v3` like `master` already in the website workflow.

Reply:
I am more in favour of reverting to v2 (also the docs) until v3 is official.